src/var/flat.js

```diff
@@ -0,0 +1,18 @@
+define( [
+	"./arr",
+	"./concat"
+], function( arr, concat ) {
+
+"use strict";
+
+// Support: IE, chrome < 69, firefox < 62, opera < 56, safari < 12
+// IE and other browsers earlier versions doesn't have `flat` method
```

Comment:
I'd phrase it differently:
```suggestion
// Provide fallback for browsers without Array#flat.
```